### PR TITLE
Detached file pointers

### PR DESCRIPTION
**Pull request by NikhilKalige**
ISIS cubefiles work for now. But the code looks at bit hackish.. Any suggestions

### CODE REVIEW

File: planetaryimage/image.py [hunk 1]
```diff
@@ -189,6 +191,11 @@ def byte_order(self):
         return '>'
 
     @property
+    def data_filename(self):
```
Comment (godber) [inline planetaryimage/image.py]: Why is this not an attribute?

**Comment by godber** [comment]
No tests?

File: planetaryimage/image.py (hunk 1 above)
Comment (NikhilKalige) [inline planetaryimage/image.py]: I just followed the existing trend :)

**Comment by godber** [comment]
Here is a `pds2isis` commands to generate some test files from our small pds3_1band.IMG.  Lets use this to get proper coverage on this merge.

```
pds2isis from=pds3_1band.IMG to=1band_tile_detached.cub+Tile+Lsb+Detached
pds2isis from=pds3_1band.IMG to=1band_bsq_detached.cub+BandSequential+Lsb+Detached
```

We still need to come up with a small PDS sample with detached label.  Converting back maybe?

**Comment by NikhilKalige** [comment]
Need to account for LINE_PREFIX_BYTES and LINE_SUFFIX_BYTES

**Comment by godber** [comment]
@NikhilKalige can you clarify your statement about LINE_PREFIX_BYTES?  Do you mean to say your commit does not do so and it is work to be done in the future?  If so, file a ticket describing the work needing to be done and preferably a test showing how it ought to work.

**Comment by godber** [comment]
I can't generate the appropriate test data to test all of these conditions.  I am sufficiently convinced that the detached label code is working for the case where the image data are offset in the file being pointed to by the `^IMAGE` pointer.  I have left a `TODO` in the test for the other cases.  I think we simply need to rely on mission data for getting our proper test coverage.

**Comment by NikhilKalige** [comment]
@godber I had problem with pds image because it has PREFIX_BYTES in the image we were using to generate the isis file. The images I used from #5 were from cassini.

**Pull request merged**